mungegithub/mungers/release-note-label.go

```diff
@@ -142,14 +142,18 @@ func (r *ReleaseNoteLabel) Munge(obj *github.MungeObject) {
 
 	labelToAdd := determineReleaseNoteLabel(obj)
 	if labelToAdd == releaseNoteLabelNeeded {
-		obj.WriteComment(releaseNoteBody)
+		if !obj.HasLabel(releaseNoteLabelNeeded) {
```

Comment:
To compare, this was the logic before:

```go
	if obj.HasLabel(doNotMergeLabel) {
		return
	}

	obj.WriteComment(releaseNoteBody)
```

So this is equivalent to the previous behavior.